driver/ppm_flag_helpers.h

```diff
@@ -9,10 +9,10 @@ or GPL2.txt for full copies of the license.
 
 #ifndef PPM_FLAG_HELPERS_H_
 #define PPM_FLAG_HELPERS_H_
+#define _GNU_SOURCE
```

Comment:
Do we need this in the header or can this be moved to a .c file? I'm not exactly comfortable with enabling _GNU_SOURCE in a single header as it means some code will be compiled with it and some without. _GNU_SOURCE e.g. changes the semantics of strerror_r slightly (while keeping the function signature) so I'd rather not enable it for random files by surprise.

Reply:
Good catch. This should be removed.